scripts/zkai_native_attention_mlp_rmsnorm_input_fused_adapter_gate.py

```
@@ -0,0 +1,544 @@
+        raise RmsnormInputFusedAdapterGateError("NANOZK win overclaim")
+
+    has_mutation_result = "mutation_result" in payload
+    provided_mutation_result = payload.get("mutation_result")
+    actual = copy.deepcopy(payload)
+    actual.pop("mutation_result", None)
+    actual.pop("payload_commitment", None)
+    expected_payload = build_payload(context, include_mutations=False)
+    expected_core = copy.deepcopy(expected_payload)
+    expected_core.pop("payload_commitment", None)
+    if actual != expected_core:
+        raise RmsnormInputFusedAdapterGateError("payload body drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)